pyrtl/helperfuncs.py

```diff
@@ -107,31 +107,8 @@ def match_bitwidth(*args):
     return (wv.zero_extended(max_len) for wv in args)
 
 
-class _ProbeIndexer(_NameIndexer):
-    """ Provides indexed probe names for probe() """
-    def __init__(self, internal_prefix='Probe'):
-        super(_ProbeIndexer, self).__init__(internal_prefix)
-        self.internal_index += 1
-
-    def next_indexed(self, w, name):
-        if not isinstance(w, WireVector):
-            raise PyrtlError('Only WireVectors can be probed')
-
-        print('(%s-%d)' % (self.internal_prefix, self.internal_index), end=' ')
-        print(get_stack(w))
-
-        if name:
-            pname = '%s%d_%s__%s)' % (self.internal_prefix, self.internal_index, name, w.name)
-        else:
-            pname = '(%s%d__%s)' % (self.internal_prefix, self.internal_index, w.name)
-
-        p = Output(name=pname)
-        p <<= w  # late assigns len from w automatically
-        self.internal_index += 1
-        return w
-
-
-probeIndexer = _ProbeIndexer()
+probeIndexer = _NameIndexer('Probe')
+probeIndexer.internal_index += 1
```

Comment:
get rid of the `internal_index += 1` . I don't care if the index starts at 0 or 1